tests/test_predefined_driven_controls.py

```diff
@@ -550,6 +551,52 @@ def test_walsh_control():
     assert np.allclose(pi_on_4_segments, _pi_on_4_segments)
 
 
+def test_gaussian_control():
+    """
+    Tests Gaussian control.
+    """
+    _rabi_rotation = np.pi
+    _segment_count = 10
+    _duration = 0.01
+    _width = 0.002
+
+    gaussian_control = new_gaussian_control(
+        rabi_rotation=_rabi_rotation,
+        segment_count=_segment_count,
+        duration=_duration,
+        width=_width,
+    )
+
+    # compute total rotation of generated pulse
+    rabi_rotation = np.dot(gaussian_control.rabi_rates, gaussian_control.durations)
+
+    _segment_width = _duration / _segment_count
+    midpoints = np.linspace(
+        _segment_width / 2, _duration - _segment_width / 2, _segment_count
+    )
+
+    def gauss(t):
+        return np.exp(-0.5 * ((t - _duration / 2) / _width) ** 2)
```

Comment:
Sorry, I committed your suggested changes to this pull request but forgot that pylint doesn't like single-letter variable names like `t`. Just change `t` to something longer like `time` and I'll be happy to approve your pull requerst.

Reply:
```suggestion
    def gauss(time):
        return np.exp(-0.5 * ((time - _duration / 2) / _width) ** 2)
```